src/containers/pages/FocusInvestigation/jurisdiction/JurisdictionList.tsx

```diff
@@ -0,0 +1,46 @@
+import { Registry } from '@onaio/redux-reducer-registry';
+import { FIReasons } from '../../../../configs/settings';
+import { ObjectList } from '../../../../helpers/cbv';
+import {
+  InterventionType,
+  makePlansArraySelector,
+  PlanFilters,
+  PlanStatus,
+} from '../../../../store/ducks/plans';
+
+/** Custom implementation of ObjectList where we override the getMapStateToProps method */
+export default class JurisdictionList<
```

Comment:
@cKellyDesign @p-netm for more complex cases like this component, you have to override one or more methods on ObjectList like I am doing here.  What are your feelings/thoughts about this?

Reply:
Seems like a pretty straight forward inheritance pattern, I love it!